Tools/Cli-LoRa-Device-Provisioning/Helpers/IoTDeviceHelper.cs

```diff
@@ -13,7 +13,8 @@ namespace LoRaWan.Tools.CLI.Helpers
 
     public class IoTDeviceHelper
     {
-        private string[] classTypes = { "A", "C" };
+        private readonly string[] classTypes = { "A", "C" };
```

Comment:
If you are using as constants I recommend changing to static. Thus preventing a new value being created every time a new IoTDeviceHelper is instanciated.

Reply:
Will do!